lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
```

Comment:
I meant to just move the file location to be consistent with the rest. (there's no good place for file-level comments on GH)